generator/internal/stub_header_generator.cc

```diff
@@ -69,13 +70,13 @@ bool GenerateClientStubHeader(pb::ServiceDescriptor const* service,
            "class $stub_class_name$ {\n"
            " public:\n");
 
-  DataModel::PrintMethods(
-      service, vars, p,
-      "  virtual grpc::Status $method_name$(grpc::ClientContext* context,\n"
-      "    $request_object$ const& request,\n"
-      "    $response_object$* response);\n"
-      "\n",
-      NoStreamingPredicate);
+  DataModel::PrintMethods(service, vars, p,
+                          "  virtual google::gax::Status $method_name$("
```

Comment:
I think I preferred the old indentation, if that is possible.

Reply:
Wrestling with clang-format is proving fruitless.